web-platform/website/src/components/pages/profile/ProfileButtons.tsx

```diff
@@ -45,11 +49,18 @@ export function ProfileButtons() {
         <Button
           className={`d-flex fw-bold ps-4 pe-4 pt-2 pb-2 ${TertiaryButton}`}
           outline
-          onClick={() => {
+          onClick={async () => {
```

Comment:
Async may not be necessary since you are not awaiting anywhere.

Reply:
yes forgot to delete it

Reply:
I wouldn't put an async without an await lol